### ISSUE TITLE
deny publicated imports

### ISSUE THREAD

**Issue by 6543** [rust-lang/rust-clippy#7953]
### What it does

It should detect if there is an import that exists multiple times (possible via import rename)

### Categories (optional)

- Kind: `clippy::correctness`?

*What is the advantage of the recommended code over the original code*

For example:
- It could lead in code publication 
- Make code more readable
- ...

### Drawbacks

None.

### Example

```rust
use std::env as environment
use std::env
```

Could be written as:

```rust
use std::env
```
